rfcs/text/0000-matrix-service.md

```diff
@@ -0,0 +1,418 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# Matrix Service
+
+## Summary
+
+The existing Matrix Service is a prototype of a Data Coordination Platform (DCP) feature that enables data consumers to perform integrative analysis across Human Cell Atlas (HCA) projects. In its current implementation, this is achieved by delivering a RESTful API that enables data consumers to generate expression matrices defined by a metadata query against an index of cells. This RFC details the prototype's major design decisions surrounding i) the Matrix Service API and ii) the service's cell-centric data model.
```

Comment:
Why is the matrix service considered a prototype feature? Should that be communicated to consumers?

Reply:
Hm the moniker is more so to communicate that the design is still up for review per this RFC.